helm/airflow/values-dev.yaml

```diff
@@ -4,6 +4,10 @@
 uid: 1002490000
 gid: 1002490000
 
+env: 
+  - name: "NAMESPACE"
+    value: "0fad32-dev"
+
```

Comment:
Where is that env variable missing? It shouln't be hardcoded, but using 
```yaml
                - name: NAMESPACE
                  valueFrom:
                    fieldRef:
                      apiVersion: v1
                      fieldPath: metadata.namespace
```

Reply:
Thanks @matthieu-foucault ! I'll update this